hopdrops/hopdrops
# fix: use Swagger integration to enable CORS

This was a huge pain in the ass, but it _should_ fix all of the CORS-related issues.

**Testing**

The workflow is a bit manual, but in order to test, invoke the following commands in the Vagrant environment:

1. `env $(cat .env) make deploy` (this deploys the API Gateway)
2. Visit http://hopdrops.io
3. Choose locations and click **Next**

If all goes well, you should see the following screen:

<img width="1392" alt="Screen Shot 2019-10-21 at 10 12 21 PM" src="https://user-images.githubusercontent.com/2184329/67255387-e00c3180-f44f-11e9-998e-81613f8a951d.png">
